e2e/lcd_test.go

```diff
@@ -0,0 +1,225 @@
+package main
```

Comment:
I was thinking to create a dedicated `cosmos/lcd` client package.
@antho1404 what do you think?

Reply:
I don't think it's really useful for now, at best we will use it for the orchestrator. I think just duplicate the call is fine.
The polling might be interesting but the querier is simple enough that we can just redo it.